docs/modules/SDDs/pages/0024-naming-scheme-v2.adoc

```diff
@@ -0,0 +1,61 @@
+= SDD 0024 - Project Syn Tools Naming Scheme v2
+
+:sdd_author:    Tobias Brunner
+:sdd_owner:
+:sdd_reviewers:
+:sdd_date:      2020-10-23
+:sdd_status:    draft
+include::partial$meta-info-table.adoc[]
+
+[NOTE]
+.Summary
+====
+Naming is hard, this page helps to make it easy. It describes the rules to name Project Syn tools.
+====
+
+== Motivation
+
+In xref:0002-naming-scheme.adoc[SDD 0002 - Naming Scheme] a naming scheme was defined for Project Syn tools which turned out to be sub-optimal.
+The naming of the tools to be non-descriptive is a high barrier of entry in to the world of Project Syn.
+This SDD describes a more descriptive naming scheme which solves this issue.
+
+=== Goals
+
+* Naming scheme which is descriptive
+* Applicable to all new tools from now on
+
+=== Non-Goals
+
+* Renaming of existing tools (like Lieutenant, Steward and Commodore) as they will change over time anyways, ultimately being replaced by newer tools.
+
+== Design Proposal
+
+`syn-$type[-$purpose]`
+
+`$type`:: Describes what kind of tool it is.
++
+Examples for types:
+
+* Operator
+* Agent
+* CLI
+* Component
```

Comment:
For components we set on the naming scheme `component-$purpose` (without the `syn-` prefix), should we really change all the components?

Reply:
I see... This SDD only applies to the tooling, not the the components. The example might be misleading.